persistence-modules/spring-mybatis/src/main/java/com/baeldung/mybatis/spring/SpringMybatisApplication.java

```diff
@@ -0,0 +1,23 @@
+package com.baeldung.mybatis.spring;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SpringMybatisApplication implements ApplicationRunner {
+
+    @Autowired
+    private ArticleMapper articleMapper;
+
+    public static void main(String[] args) {
+        SpringApplication.run(SpringMybatisApplication.class);
+    }
+
+    @Override
+    public void run(ApplicationArguments args) throws Exception {
+        articleMapper.getArticle(1L);
```

Comment:
It's more conventional to write unit tests rather than `main` programs you run here.